### PR TITLE
Corrections made to lesson 2 assignments

### PR DESCRIPTION
**Pull request by jayjonesseattle**
I have made the suggested corrections to the fizzbuzz.py and series.py assignments. I will use an IDE moving forward to prevent the indenting errors if that is okay

### CODE REVIEW

File: students/terrance_jones/lesson2/series.py [hunk 1]
```diff
@@ -60,16 +73,20 @@ def sum_series(n, a=0, b=1):
...
+			return True
+		else:
+			return False
+
+
+
+if __name__ == '__main__':
+	print("testing")
+	print()
+	assert fibonacci(6) == True
```
Comment (natasha-aleksandrova) [inline students/terrance_jones/lesson2/series.py]: from instructions 

> The function should return the nth value in the fibonacci series (starting with zero index)


so it should be `assert fibonacci(3) == 2`

**Comment by natasha-aleksandrova** [comment]
Yes I recommend using an IDE like Pycharm, which is great for improving code style and catch violations early.

**Comment by jayjonesseattle** [comment]
Thank you for your feedback. I didn’t quite understand the asset part of
the exercise but you helped make it clear. I apologize for not correcting
the indentation. I totally forgot. I will make the corrections and push
them later today. Thank you!

**Comment by natasha-aleksandrova** [comment]
sounds good!

**Pull request merged**